### PR TITLE
[stdlib] Formats MoveStdlib package 1/N

### PR DESCRIPTION
**Pull request by damirka**
## Description 

Uses module label in the MoveStdlib, runs the current build of the formatter on the code.

## Test plan 

All tests must pass, no snapshot regen should be required.

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

**Review by amnn — APPROVED** [review]
Seems reasonable!

File: crates/sui-framework/packages/move-stdlib/sources/type_name.move [hunk 1]
```diff
@@ -2,58 +2,58 @@
...
-    public native fun get_with_original_ids<T>(): TypeName;
-
-    /// Returns true iff the TypeName represents a primitive type, i.e. one of
-    /// u8, u16, u32, u64, u128, u256, bool, address, vector.
-    public fun is_primitive(self: &TypeName): bool {
-        let bytes = self.name.as_bytes();
-        bytes == &b"bool" ||
+module std::type_name;
+
+use std::{address, ascii::{Self, String}};
```
Comment (tnowacki) [inline crates/sui-framework/packages/move-stdlib/sources/type_name.move:7]: not sure I like the `use` grouping here. Strong feelings? If not, maybe revert for this PR

**Review by tnowacki — APPROVED** [review]
Fabulous! My eyes thank you

File: crates/sui-framework/packages/move-stdlib/sources/vector.move [hunk 2]
```diff
@@ -4,368 +4,371 @@
...
-        while (i < len) {
-            v.swap(i, len);
-            i = i + 1
-        }
-    }
+    len = len - 1;
+    while (i < len) v.swap(i, {
+            i = i + 1;
+            i
+        });
```
Comment (tnowacki) [inline crates/sui-framework/packages/move-stdlib/sources/vector.move:134]: Was this auto formatted? If so, its a good bug to fix IMO :)

File: crates/sui-framework/packages/move-stdlib/sources/type_name.move (hunk 1 above)
Comment (damirka) [reply crates/sui-framework/packages/move-stdlib/sources/type_name.move]: It's what formatter does these days.

File: crates/sui-framework/packages/move-stdlib/sources/vector.move (hunk 2 above)
Comment (damirka) [reply crates/sui-framework/packages/move-stdlib/sources/vector.move]: Ty!

**Review by cgswords — COMMENTED** [review]
🎉

**Pull request merged**